example/instrum/Makefile.am

```diff
@@ -0,0 +1,20 @@
+LIB   = $(top_builddir)/lib
+
+AM_CFLAGS = \
+        -I$(srcdir) \
+        -I$(top_srcdir)/example \
+        -I$(top_srcdir)/platform/@with_platform@/include \
+        -I$(top_srcdir)/include/ \
+        -I$(top_srcdir)/include/odp/arch/@ARCH_ABI@ \
+        -I$(top_srcdir)/helper/include \
+        -I$(top_builddir)/platform/@with_platform@/include \
+        -I$(top_srcdir)/platform/@with_platform@/arch/@ARCH_DIR@ \
+        -I$(top_builddir)/include \
+        -I$(PAPI_PATH)/include
```

Comment:
Please restructure include flags:
 - Use `$(ODP_INCLUDES)` to get all ODP includes
 - Use `$(PAPI_CFLAGS)` here.